flexget/components/sites/sites/lostfilm.py

```diff
@@ -206,7 +210,14 @@ def on_task_input(self, task, config):
             try:
                 response = task.requests.get(redirect_url, params=params)
             except RequestException as e:
-                logger.error('Failed to get lostfilm download page: {:s}'.format(e))
+                logger.error('Failed to get the redirect page: {}', e)
+                continue
+            except (CloudflareException, CaptchaException) as e:
```

Comment:
Hmm, won't this raise a `NameError` when cloudscraper isn't installed?

Reply:
If you want those to be conditional I think something like this can work.
```python
try:
    from cloudscraper.exceptions import CloudflareException, CaptchaException
    cloudscraper_exceptions = (CloudflareException, CaptchaException)
except ModuleNotFoundError:
    cloudscraper_exceptions = ()
...
try:
    blah()
except cloudscraper_exceptions as e:
    do_something()
```

Reply:
No, it's not needed. I've tried without cfscraper - no problem.
You can try by adding any `except SomeNonExistingExpectionError` and it wouldn't rise any additional exceptions.

Reply:
It will if there is an exception though.
```python
try:
    raise Exception()
except SomeNonExistingExceptionError:
    pass

Traceback (most recent call last):
  File "<input>", line 2, in <module>
Exception

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<input>", line 3, in <module>
NameError: name 'SomeNonExistingExceptionError' is not defined
```

Reply:
Hmm, seems exception wasn't raised when I tested it.
I'll fix it.

Reply:
Updated